src/confdb/confdb.c

```diff
@@ -1549,6 +1554,10 @@ static errno_t confdb_init_domain_subdomains(struct sss_domain_info *domain,
             goto done;
         }
     }
+#else
+    split_on_separator(domain, "none", ',', true, true,
```

Comment:
Shouldn't we check the return value of `split_on_separator()`?

Reply:
Fixed.